libraries/botbuilder-dialogs-adaptive-testing/src/settingMocks/settingMock.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * @module botbuilder-dialogs-adaptive-testing
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Configurable, Converter } from 'botbuilder-dialogs';
+import { ResourceExplorer } from 'botbuilder-dialogs-declarative';
+
+/**
+ * Abstract class for mocking settings.
+ */
+export abstract class SettingMock extends Configurable {}
+
+/**
+ * The type converters for SettingMock.
+ */
+export class SettingMocksConverter implements Converter<string[], SettingMock[]> {
+    public constructor(private readonly _resourceExplorer: ResourceExplorer) {}
+
+    public convert(value: string[] | SettingMock[]): SettingMock[] {
+        const settingMocks: SettingMock[] = [];
+        value.forEach((item: string | SettingMock) => {
```

Comment:
I think `map` is probably a touch more performant in general and better captures what the code is doing.